wl_menu.c

```diff
@@ -340,7 +340,7 @@ static byte *ExtScanNames[] =   // Names corresponding to ExtScanCodes
                                         };*/
 
 #else
-static const char* const ScanNames[SDLK_LAST] =
+/*static const char* const ScanNames[SDLK_LAST] =
```

Comment:
I am not sure what to do with these preprocessor controlled blocks that had the old ScanNames arrays in them. SDL2 has gaps in its key ints starting with [capslock ](https://wiki.libsdl.org/SDLKeycodeLookup) which is why I moved to switch statements to convert to indexes.